server/eb/app.py

```diff
@@ -0,0 +1,53 @@
+"""cellxgene AWS elastic beanstalk application"""
+
+import sys
+import os
+import logging
+
+logging.basicConfig(
+    filename="/opt/python/log/app.log",
```

Comment:
Question on logging:  will this also put the info into the web logs so it is visible from the EB/AWS console?  Or does this only drop it into the file?   Naively, I assumed we would want to see the logs in the default console/tooling.

Reply:
Logging goes to both /opt/python/log/app.log and stdout.
The next statement down has the handling for stdout.
/opt/python/log/app.log  is the magic location that makes it visible (e.g. 'eb logs' will find it).
I'll add a check that /opt/python/log exists before adding this line, so that you can run the app locally without problems.

Reply:
Out of curiosity, what is the value of also logging into a file?  Will deployments need to add log rotation, etc?

I'm fine leavings as-is for this interim PR.

Reply:
I am doing it both to a file and stdout for some convenience.  I was able to test my application locally and catch error more quickly that way when I could see the logs in stdout.  But for running in EB, the logs need to go to that file.    We will probably want to add log rotation too.

Reply:
sounds good.  I suspect much of this other configuration & deployment tooling belongs outside the cellxgene repo.  Thanks for the detailed response.